README.md

```
@@ -0,0 +1,135 @@
+
+| Category  | Choice                                                                                                        |
+| --------- | ------------------------------------------------------------------------------------------------------------- |
+| Framework | ASP.NET Core (.NET 10)                                                                                        |
+| Database  | PostgreSQL 17                                                                                                 |
+| ORM       | EF Core + Npgsql (planned)                                                                                    |
+| API docs  | OpenAPI + [Scalar](https://github.com/scalar/scalar) UI (`Microsoft.AspNetCore.OpenApi`, `Scalar.AspNetCore`) |
+| Container | Docker + Docker Compose                                                                                       |
+
+## Project structure
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)